### PR TITLE
Feature/button theme table

### PR DESCRIPTION
**Pull request by DanSallau**
The primary button should have theme

### CODE REVIEW

File: src/Table/Table.tsx [hunk 1]
```diff
@@ -228,7 +230,9 @@ const ActionColumn: React.FunctionComponent<ActionColumnProps> = (props: ActionC
                 <button
                     id={btnPrimaryRandomIds}
                     type="button"
-                    className="btn btn-outline-primary btn-sm"
+                    className={`btn btn-${props.primaryActionButton.buttonTheme ? props.primaryActionButton.buttonTheme : "outline-primary"} btn-${
+                        props.primaryActionButton?.buttonSize ? props.primaryActionButton?.buttonSize : "sm"
```
Comment (mario-subo) [inline src/Table/Table.tsx:234]: the `buttonSize` is defined s `"btn-lg" | "btn-md" | "btn-sm"` but if I pass `"btn-lg"` as the `buttonSize` prop the className will result in`"btn-btn-lg"`

File: src/Table/Table.tsx [hunk 2]
```diff
@@ -53,6 +53,8 @@ export interface ActionLinkItem {
 
 export interface PrimaryActionButton {
     label: string;
+    buttonTheme?: "link" | "outline-primary" | "secondary" | "ghost-dark" | "ghost-light" | "danger" | "primary";
+    buttonSize?: "btn-lg" | "btn-md" | "btn-sm";
```
Comment (mario-subo) [inline src/Table/Table.tsx:57]: I guess you have to remove the `btn-` here or don't append it in the className

Comment (DanSallau) [reply src/Table/Table.tsx]: Oh shit! I guess this is why unit test is important.

**Pull request merged**